cardsGame/Deck.java

```diff
@@ -0,0 +1,55 @@
+package com.asad.cardsGame;
+
+import java.util.Random;
+
+public class Deck {
+	private Card[][] deck;
+
+	public Deck() {
+		this.deck = new Card[4][13];
+		this.deck = buildADeck(this.deck);
+	}
+
+	/**
+	 * building a Deck of 52 cards
+	 * 
+	 * @param deck is a 2D array of cards
+	 * @return array of 52 cards
+	 */
+	private Card[][] buildADeck(Card[][] deck) {
+		for (int i = 0; i < deck.length; i++) {
+			for (int j = 0; j < deck[i].length; j++) {
+				deck[i][j] = new Card(i, j);

```

Comment:
new Card(i + Card.DIAMOND, j+Card.TWO) is better - because this constant may change !